src/components/look-controls.js

```diff
@@ -336,36 +339,36 @@ module.exports.Component = registerComponent('look-controls', {
   /**
    * Register touch down to detect touch drag.
    */
-  onTouchStart: function (evt) {
-    if (evt.touches.length !== 1 ||
+  onTouchStart: function (event) {
+    if (event.touches.length !== 1 ||
         !this.data.touchEnabled ||
         this.el.sceneEl.is('vr-mode')) { return; }
     this.touchStart = {
-      x: evt.touches[0].pageX,
-      y: evt.touches[0].pageY
+      x: event.touches[0].pageX,
+      y: event.touches[0].pageY
     };
     this.touchStarted = true;
   },
 
   /**
    * Translate touch move to Y-axis rotation.
    */
-  onTouchMove: function (evt) {
```

Comment:
Can we keep `evt` instead of `event` for consistency with rest of the code base?